src/views/ProtestPage.js

```diff
@@ -203,7 +193,7 @@ export default function ProtestPage({ user }) {
 
   const { coordinates, id } = protest;
 
-  const canEdit = isAdmin(user) || isLeader(user, protest);
+  const canEdit = !!user && (user?.edits?.length || 0) < 5; //isAdmin(user) || isLeader(user, protest);
```

Comment:
I know that it's not relevant specifically to here, and that it's not critical. But the name "edits" doesn't explain that this is an array of unique protest Ids, and one might think that those are individual edits.

It worth either change the name of this property to something like `edited_protests` or add a comment about who can edit, so this logic will be a bit clearer.